shared/core/types/proto/relay.proto

```diff
@@ -6,24 +6,37 @@ option go_package = "github.com/pokt-network/pocket/shared/core/types";
 
 // TECHDEBT: These structures were copied as placeholders from v0 and need to be updated to reflect changes in v1
 message Relay {
-    RelayPayload payload = 1;
-    RelayMeta meta = 2;
+    RelayMeta meta = 1;
+    oneof RelayPayload {
+        JSONRPCPayload json_rpc_payload = 2;
+        RESTPayload rest_payload = 3;
+        //DISCUSS: content of the following types:
+        // GRPCPayload grpc_payload = 3;
+        // GraphQLPayload graphql_payload = 4;
+        // WebSocketsPayload websockets_payload = 5;
+    }
 }
 
-message RelayPayload {
-    string data = 1;
-    string method = 2;
-    string http_path = 3;
-    map<string, string> headers = 4;
+message RESTPayload {
+    string contents = 1;
+}
+
+message JSONRPCPayload {
+    bytes id = 1;
+    string json_rpc = 2;
+    string method = 3;
+    bytes parameters = 4;
```

Comment:
should this be `repeated bytes parameters`?

Reply:
Not based on my understanding of the `parameters` field in [JSONRPC spec](https://www.jsonrpc.org/specification#parameter_structures). I have updated the proto file with a comment on this. Please let me know whether that addresses this review item.